src/ui/gradio_ui.py

```diff
@@ -36,8 +36,10 @@ def chat_ui(self, prompt, history, use_history=None):
         json_data = json.dumps(data)
 
         try:
-            # Make the HTTP POST request
-            response = requests.post(self.ols_url, headers=headers, data=json_data)
+            # Make the HTTP POST request, wait for response with 10 seconds timeout
+            response = requests.post(
+                self.ols_url, headers=headers, data=json_data, timeout=10
```

Comment:
removing my lgtm.  This needs to be higher.  The LLM generation calls can be pretty slow.  Make it 30s.

Reply:
okay

Reply:
updated